### DISCUSSION TITLE
Minor docker tags should be using lastest release

### DISCUSSION THREAD

**Discussion by anbraten** [woodpecker-ci/woodpecker#2154] [General]
We got quite some confusion in the last days about the 1.0 docker tags. Instead of building the newest version of the release branch I would suggest to always push the last tagged release of that version as its done by most other projects.

**Comment by pat-s** [comment]
Suggestion:

- `v1` -> latest state of the release branch, rolling, build every day
- `v1.x` -> fixed state, not updating

**Reply by anbraten** [reply]
I think most users expect `v1` and `v1.x` to be tagged "stable" releases. If we need / want to push images of the release branch we could maybe consider sth like `v1-next`

**Reply by pat-s** [reply]
`v1` being a rolling tag is quite common - all GitHub Actions work like that and Gitea also does it (and probably more).

I don't like `v1-next`, I'd argue that this is quite uncommon and would lead to confusion. 

> I think most users expect v1 and v1.x to be tagged "stable" releases

They are, aren't they? `v1` is just the most recent release of the v1 series - without users having to explicitly change their image tag.

**Comment by smainz** [comment]
My expectation was that `v1.0` always points to the latest `v1.0.x` and `v1` always points to the latest `v1.x.`
So i get the latest release of one of these.

I went back from `v1.0` to `v1.0.1` when I saw that `v1.0` identifies itself as `next-<something>` 
But this is not ideal, as I do not get patches automatically.

**Comment by qwerty287** [comment]
That's outdated as we don't do backports anymore and don't have release branches
